app/components/list/bulk-actions/bulk-list-header.tsx

```diff
@@ -0,0 +1,56 @@
+import { useLoaderData } from "@remix-run/react";
+import { useAtomValue, useSetAtom } from "jotai";
+import {
+  selectedBulkItemsCountAtom,
+  setSelectedBulkItemsAtom,
+} from "~/atoms/list";
+import { FakeCheckbox } from "~/components/forms/fake-checkbox";
+import { PartialCheckboxIcon } from "~/components/icons/library";
+import { Th } from "~/components/table";
+import type { IndexResponse } from "~/routes/_layout+/assets._index";
+import { ALL_SELECTED_KEY } from "~/utils/list";
+import { tw } from "~/utils/tw";
+
+export default function BulkListHeader() {
+  const { items } = useLoaderData<IndexResponse>();
+
+  const setSelectedBulkItems = useSetAtom(setSelectedBulkItemsAtom);
+  const itemsSelected = useAtomValue(selectedBulkItemsCountAtom);
+
+  const partialItemsSelected =
+    itemsSelected > 0 && itemsSelected < items.length;
+
+  const allItemsSelected = itemsSelected >= items.length;
```

Comment:
This is a bit confusing for me. I see the code for `alItemsSelected` and i assume this refers to what we discussed with the supabase example. Being able to select everything, not only from current page.
So i see this code but I don't see a button to do it.